modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -9,7 +12,8 @@
         zIndex: 2000,
         animation: 'fade',
         inertia: true,
-        placement: 'bottom'
+        placement: 'bottom',
+        hideOnClick: true
```

Comment:
This is the best compromise I found to fix the fact that when double clicking on the bubble menu (but not on a button or select), the bubble menu wasn't closing when unfocusing the richtext area (a weird issue the the bubble menu div being focused).
See [this issue](https://github.com/ueberdosis/tiptap/issues/1433) for more info.